emgapianns/management/commands/import_analysis.py

```diff
@@ -355,15 +367,17 @@ def __find_folder(self, directory, search_pattern, maxdepth=2, recursive=False):
 
         if study_folder:
             if len(study_folder) > 1:
-                sys.exit(f'Found more than 1 result directory: {study_folder}')
+                logging.info(f'Found more than 1 result directory: {study_folder}')
+                return study_folder
             else:
                 logging.info(f'Found result dir: {study_folder}')
                 return study_folder[0]
```

Comment:
this should be a list too, otherwise this method will return a list or a path (string) sometimes.